docs/internals/contributing/writing-documentation.txt

```diff
@@ -45,11 +45,12 @@ documentation is transformed into HTML, PDF, and any other output format.
 __ https://www.sphinx-doc.org/
 __ https://docutils.sourceforge.io/
 
-To build the documentation locally, install Sphinx:
+To build the documentation locally, install the doc requirements from the
+project root
```

Comment:
Same suggestion as before, given that the next command is executed from within the `docs/` folder, I would place the reader in that folder form the beginning. Bonus point since some of us can just `ctrl+r` into the `python -m pip install -r requirements.txt` bash history line!